packages/autop/src/test/index.test.js

```diff
@@ -134,7 +134,7 @@ Paragraph two.`;
 Paragraph two.`;
 
 	const expected =
-		'<p>Paragraph one.</p>\n' + // line breaks only after <p>
+		'<p>Paragraph one.</p>\n' + // Line breaks only after <p>
```

Comment:
Do we have to add a dot at the end of this line?

Reply:
I'm not sure, I purposely stopped it from considering lines that ended in non-word characters, do you think I should remove this exception and make periods mandatory even after non-word chars?

Reply:
I think it's fine to leave it as is. The simple solution would bo to end a sentence with a word 😄

Reply:
Well, can we still fix it manually without conflicting with the rule fixer?